Dockerfile.s390x

```diff
@@ -48,6 +48,21 @@ RUN apt-get update && apt-get install -y \
 	tar \
 	--no-install-recommends
 
+# install seccomp: the version shipped in jessie is too old
+ENV SECCOMP_VERSION 2.3.0
+RUN set -x \
+	&& export SECCOMP_PATH="$(mktemp -d)" \
+	&& curl -fsSL "https://github.com/seccomp/libseccomp/releases/download/v${SECCOMP_VERSION}/libseccomp-${SECCOMP_VERSION}.tar.gz" \
+		| tar -xzC "$SECCOMP_PATH" --strip-components=1 \
+	&& ( \
+		cd "$SECCOMP_PATH" \
+		&& ./configure --prefix=/usr/local \
+		&& make \
+		&& make install \
+		&& ldconfig \
+	) \
+	&& rm -rf "$SECCOMP_PATH"
+
```

Comment:
I discussed the issue with Srini: We would like to _leave the PR as it is_ because it will test if runc is built properly with libseccomp on s390. At the moment we _don't want to enable seccomp in the DOCKER_BUILDTAGS_ because it looks like there are also some issues with Docker and seccomp on s390 which we first want to check.

I rebased the PR. It can be merged now.